ext/storage_sources/azure_store/test/test_azure_connection.cpp

```diff
@@ -31,4 +31,4 @@
 
 #include "azure_connection.h"
 
-TEST_CASE("testing class azure_connection", "azure_connection") {}
+TEST_CASE("Testing Class azure_connection", "azure_connection") {}
```

Comment:
Please revert this file.

Reply:
done in https://github.com/wiredtiger/wiredtiger/pull/8738/commits/a45f499619cc9f05c718cbc683ae2f59f48b8bd0